### DISCUSSION TITLE
When do you plan to release Micronaut 4.0.0 Relese Candidate / beta?

### DISCUSSION THREAD

**Discussion by alapierre** [micronaut-projects/micronaut-core#8866] [General] [answered]
I'm waiting for some new features in Micronaut Data 4.0.0 which will be available from 4.0.0. If I know, that you need eg. a month for release, I will find some bypass

**Comment by graemerocher** [comment, marked as answer]
We are almost feature complete. A milestone will be released in the next couple of weeks and then we will release regular milestones and RCs with the goal of going GA later in Q2

**Reply by alapierre** [reply]
Thank you. I will test regularly SNAPSHOTS. If I can help in any way, just let me know.
